ansible_collections/arista/avd/roles/eos_designs/python_modules/overlay/router_bgp.py

```diff
@@ -274,7 +274,7 @@ def _address_family_evpn(self) -> dict:
         if self.shared_utils.overlay_dpath is True:
             address_family_evpn["domain_identifier"] = get(self.shared_utils.switch_data_combined, "ipvpn_gateway.evpn_domain_id", default="65535:1")
 
-        if self.shared_utils.is_wan_server:
+        if self.shared_utils.is_wan_server and not get(self._hostvars, "wan_route_servers_next_hop_resolution", default=False):
```

Comment:
We need to reconsider some of the evpn config on the pathfinders and maybe tie it to this knob. Why do we export the loopback0 if we cannot respond to a packet.